tests/network/localnet/test_default_bridge.py

```diff
@@ -33,3 +40,20 @@ def test_connectivity_post_migration_between_localnet_vms(migrated_localnet_vm,
         port=8888,
     ) as (client, server):
         assert is_tcp_connection(server=server, client=client)
+
+
+@pytest.mark.ipv4
+@pytest.mark.single_nic
+@pytest.mark.s390x
+@pytest.mark.usefixtures("nncp_localnet")
+@pytest.mark.polarion("CNV-12363")
+def test_vmi_reports_ip_on_secondary_interface_without_vlan(
+    vm_localnet_1,
+    vm_localnet_1_primary_secondary_ips,
+):
+    """
+    Test that vm_localnet_1's secondary interface on a no-VLAN localnet
+    correctly reports the IP address for that interface.
+    """
+    _, secondary_ip = vm_localnet_1_primary_secondary_ips
+    assert get_vmi_ip_v4_by_name(vm=vm_localnet_1, name=LOCALNET_BR_EX_NETWORK_NO_VLAN) == ip_interface(secondary_ip).ip
```

Comment:
Will `lookup_iface_status` fit here?

Reply:
`lookup_iface_status` fetches an interface status from a given VM.
I find it trivial that from the information received, an assertion can be done on one of its values.

Reply:
May I resolve this thread?